mephisto/abstractions/architects/ec2/ec2_architect.py

```diff
@@ -52,6 +52,16 @@ def url_safe_string(in_string: str) -> str:
     return re.sub("[^0-9a-zA-Z-]+", "", hyphenated)
 
 
+def get_full_domain(args: DictConfig) -> str:
```

Comment:
Is there a specific use for extracting this out? The architect still does this in `__init__`, but I see where knowing this information without setting up an `__init__` could be useful. If so, just add a comment for the function and then use it in the `__init__` call below.

Edit: found the usage, I don't think it's necessary to have used in the place it is, but the util could still be useful.